webSites/webSites-war/src/main/java/org/silverpeas/components/websites/servlets/design/SiteDesignActionHandler.java

```diff
@@ -0,0 +1,660 @@
+    }
+
+    // ADD NEW SITE -------------------------------------------------------------
+    if ("newSite".equals(action)) {
+      currentPath = addNewDesignedSite(controller, request);
+    } else if ("updateDescription".equals(action)) { // type 0 design ou 2 upload
+      updateDesignedSite(controller, request, currentPath);
+    } else if ("addFolder".equals(action)) {
+      addFolderInSite(controller, request, currentPath);
+    } else if ("renameFolder".equals(action)) {
+      renameFolderInSite(controller, request, currentPath);
+    } else if ("deleteFolder".equals(action)) {
+      deleteFolderInSite(controller, request, currentPath);
+    } else if ("addPage".equals(action)) {
+      addPageInSite(controller, request, currentPath);
+    } else if ("renamePage".equals(action)) {
+      renamePageInSite(controller, request, currentPath);
+    } else if ("deletePage".equals(action)) {
+      deletePageInSite(controller, request, currentPath);
+    } else if ("classifySite".equals(action)) { // cas de l'upload et du design
+      classifySite(controller, request, id);
+    } else if ("design".equals(action)) {
+      // DESIGN -------------------------------------------------------------
+      SiteDetail site = controller.getWebSite(id);
+      controller.setSessionSite(site);
+      request.setAttribute("Site", site);
+    } else {
+      // AUTRE -------------------------------------------------------------
+      // view en cas de rechargement de la page pour naviguer dans le chemin
+      // ou createSite annule
+      // ou upload d'image
+      SiteDetail site = controller.getSessionSite();
+      id = site.getPK().getId();
+      request.setAttribute("Site", site);
+    }
+
+    return "/webSites/jsp/design.jsp?Action=design&path=" + currentPath + "&Id=" + id;
+  }
+
+  private void classifySite(final WebSiteSessionController controller, final HttpRequest request,
+      final String id) throws WebSitesException {
+    // CLASSIFY SITE -------------------------------------------------------------
+    // = en cas de new Site ou de classifySite
+    String listeTopics = request.getParameter(WebSitesRequestRouter.TOPIC_LIST_PARAM);
+
+    request.setAttribute("Site", controller.getSessionSite());
+
+          /* publications : classer le site dans les themes cochés */
+    ArrayList<String> arrayToClassify = new ArrayList<>();
+    boolean publish = false;
+
+    ArrayList<String> arrayTopic = new ArrayList<>();
+    int begin = 0;
+    int end = 0;
+    end = listeTopics.indexOf(',', begin);
+    String idTopic = null;
+    while (end != -1) {
+      idTopic = listeTopics.substring(begin, end);
+
+      begin = end + 1;
+      end = listeTopics.indexOf(',', begin);
+
+      arrayTopic.add(idTopic);
+      publish = true;
+    }
+    controller.updateClassification(id, arrayTopic);
+
+    arrayToClassify.add(id);
+    if (publish) {
+      controller.publish(arrayToClassify); // set etat du site a 1
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3ATrailingCommentCheck)